java/src/jmri/jmrit/dispatcher/AutoAllocate.java

```diff
@@ -102,7 +105,7 @@ protected synchronized void scanAllocationRequestList(List<AllocationRequest> li
         copyAndSortARs(list);
         removeCompletePlans();
         for (int i = 0; i < orderedRequests.size(); i++) {
-            try {
+            //try {
```

Comment:
The removal of this try/catch will cause the AutoAllocate process to jam in the event of an err, particully the erro caused when a terminating train thread, cleans up at the same time as an scan is taking place.

Reply:
The try catch should be in place. I thought I had reinstated it with another commit. I have checked that it is in place in my latest Dispatcher3 code

Reply:
I commented out the try catch to resolve an exception and debug the code.

Reply:
I reinstated with a commit last night. Let me know if there are any problems